src/pages/AddItem.js

```diff
@@ -3,39 +3,42 @@ import { withFirestore } from 'react-firestore';
 import AddItemHeader from './AddItemHeader';
 import getToken from '../lib/token';
 
-const expectedPurchase = { soon: 7, kindOfSoon: 14, notSoon: 30 };
-
 const AddItem = ({ firestore }) => {
   const [name, setName] = useState('');
 
-  const [nextExpectedPurchase, setNextExpectedPurchase] = useState(0);
-
   const initialToken = () => window.localStorage.getItem('token') || getToken();
   const [token] = useState(initialToken);
+  // const[ redirect, setRedirect ] = useState(false)
   useEffect(() => {
     window.localStorage.setItem('token', token);
   }, [token]);
 
+  console.log('token:', token);
+
+  // var query = firestore.collection('items').where('token', '==', token);
+  // console.log(query)
+  // Send the new item to Firebase
   const addItem = name => {
-    firestore.collection('items').add({ name, token, nextExpectedPurchase });
+    firestore.collection('items').add({ name, token });
```

Comment:
The joining of the list works, but why is this part taken out?

Reply:
@MonicaDJohnson @ajiles91 

everything looks good but like @prophen mentioned what was the reason for taken out  `nextExpectedPurchase` and the form?

Reply:
I think it got taken out accidentally when we were trying to create the form to enter the token. I just added it back, and pushed the change to our branch. My apologies!